Spec.md

```diff
@@ -479,7 +479,18 @@ only.
 A list of mods which *conflict* with this mod. The current mod
 *will not* be installed if any of these mods are already on the system.
 
-##### resources
+##### replaced-by
+
+(**v1.24**) This is a way to mark a specific mod identifier as being
+obsoleted and tell the client what it has been *replaced by*. It contains a
+single mod that should be selected for installation if this mod is
+selected for updating, while this mod is uninstalled. If this mod identifier
+is brought back to life, an epoch change should be applied. A *replaced_by*
+relationship should be added to the final release of the mod being replaced.
+The listed mod should include a "provides" relationship either to this mod,
```

Comment:
The stuff about "provides" makes this sound complicated, since now I have to worry about two changes to two different mods. Will `replaced_by` work on its own? What happens if the provides relation is missing?

Reply:
Also, does this advice apply to the multiple-replacement case? Presumably if a mod only replaces part of another mod, it would not be correct to say that it "provides" that mod.

Reply:
There is no multiple-replacement case. We are only permitting a singe replacement mod.

Replaced_by works on its own. This is a "should", not a "must".